### ISSUE TITLE
Accessibility: governance reporting guide

### ISSUE THREAD

**Issue by Bennyonye** [jupyterhub/outreachy#118]
### What page is this for?

https://jupyter.org/try-jupyter/retro/notebooks/

### WAVE accessibility report

![Screenshot 2022-10-23 192429](https://user-images.githubusercontent.com/98492400/197409263-e70142c5-5da1-4a75-a65c-58ce6fa5cd9d.jpg)

9 Errors
Missing or uninformative page title
Language missing or invalid

0 Contrast Error

0 Alerts

3 Features

26 Structural Element

8 ARIA

### The HTML element to be changed

```html
1. <html> without attribute `lang`
2. 5X Empty form label
    <label for="toctree-checkbox-2">
      <i class="fas fa-chevron-down"></i>
    </label>

3. 2X Multiple form labels
   <input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">

4. 1X Empty button
   <button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" 
      title="" data-original-title="Fullscreen mode">
      <span class="headerbtn__icon-container">
         <i class="fas fa-expand"></i>
      </span>
   </button>
5. 1X Linked image with alternative text
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">

6. 3X Heading level 1
   3X Heading level 2

7. 14X Unordered list
```

### Your proposed HTML (or CSS), after change

```html
1. include HTML lang attribute: i.e. 
     <html lang="en"></html>
2. <label for="__navigation" class="headerbtn">Search</label>
3. <div class="input-wrapper">
     <label for="__navigation" class="overlay overlay-navbar">Search all</label>

     <input type="checkbox" id="__navigation" name="q"> 
  </div>
4. <button type="submit">
     <img src="/search-icon.svg" alt="Search"/>
   </button>
                      or
  <input type="button" value="button">

5.<a href="https://demosite.equalizedigital.com/?page_id">
  <img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">

6.<h1></h1>
  <h2></h2>

7.Unordered list
   <ul class="nav bd-sidenav bd-sidenav__home-link">
    <li class="toctree-l1">
      Project Jupyter Governance
     </a>
  </ul>
```

### More info

> When a screen reader loads a web page, the text within the 'title' is the first thing it sees. If the title does not describe what is on the page, readers who rely on HTML markup will have a difficult time determining which page they are on. The webpage 'title' is added to the page 'head' to assist readers in understanding the specific page they are accessing.

> To fix empty or missing form label errors, you’ll need to determine how the field and form were created and then either add text to your existing field label or add field labels if they are missing completely. For any pages or posts that have either an Empty Form Label error or a Missing Form Label error in the WordPress editor, you can open the details tab in the Accessibility Checker meta box, then expand the applicable error to see a list of code that caused the error to appear.

> To fix duplicate form label errors, you’ll need to determine how the field and form were created and then ensure that each field has only one label associated with it. Once determined, you need to edit the form or HTML code so that there is only one label. If you’re using a form building plugin that has incorrectly added two labels to a particular type of field, you can typically resolve the error by stop using that field type and replacing it with a different field type that will accept the same information.

> For any pages or posts that have an Empty Button error in the WordPress editor, you can open the details tab in the Accessibility Checker meta box, then expand the Empty Button error to see a list of code that caused the error to be flagged. Then resolve the error by stop using that field type and replacing it with a different field type that will accept the same information.

> To fix this error, you will need to add meaningful alt text to the images that flagged the Linked Image Missing Alternative Text or Linked Image Empty Alternative Text error, ensuring that all linked images have alt text that describes the function of the link.

>  In order for a link to be readable by screen reader users links must all contain either plain text, or in the case of a linked image, alt text describing where the link goes. If the linked image contains text, a good general rule is to make the alt text the same, or similar to the text in the image. Screen reader support for reading the title attribute for descriptions of images is inconsistent and should not be relied upon as the sole means to convey important information. However, it is alright to use the title attribute in conjunction with an alt attribute.

> There are five levels of heading in APA Style. Level 1 is the highest or main level of heading, Level 2 is a subheading of Level 1, Level 3 is a subheading of Level 2, and so on through Levels 4 and 5. The number of headings to use , depends on the length and complexity of the work. 
The level 1 heading is the most important header in the document, It should be rendered more prominently than any other header. It is usually used to indicate the title of the document often it has the same contents as the [TITLE], although this is not required and not always a good idea. The title should be useful out of context (for example, in a bookmarks file) but the level 1 heading is only used inside the document. For subsections within the introduction, use Level 2 headings for the first level of subsection

**Comment by minrk** [comment]
Thanks for these suggestions! Can you be more specific in the suggestions for 6 and 7 and why - actual, exact HTML elements to change, not just the _kind_ of changes to be made? The goal here is to apply the principles to specific changes, after identifying the relevant principles.

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
